nanopub/publication.py

```diff
@@ -97,6 +97,9 @@ def from_assertion(cls, assertion_rdf, uri=DEFAULT_NANOPUB_URI, introduces_conce
 
         # Set up different contexts
         rdf = rdflib.ConjunctiveGraph()
+        # Use namespaces from assertion_rdf
+        for prefix, namespace in assertion_rdf.namespaces():
```

Comment:
Kind of surprised ```rdflib``` doesn't automatically incorporate bindings from graphs when they are added. Nevermind...

Reply:
Yeah, I was also surprised, because I thought we looked at this together and concluded that. But tested it a few times and you really need this. 

Indeed this is a general fix.